service/src/test/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCacheTest.java

```diff
@@ -370,7 +368,7 @@ public void testGetWithResetForSubjectsByIds() throws Exception {
         assertEquals(cachedResult.getEffect(), result.getEffect());
 
         Thread.sleep(1);
-        this.cache.resetForSubjectsByIds(ZONE_NAME, new HashSet<>(Arrays.asList(AGENT_MULDER)));
+        this.cache.resetForSubjectsByIds(ZONE_NAME, new HashSet<>(Collections.singletonList(AGENT_MULDER)));
```

Comment:
Use `Collections.singleton(..)` instead of `new HashSet<>(Collections.singletonList(..))`